### DISCUSSION TITLE
How to launch playlist from command line

### DISCUSSION THREAD

**Discussion by jan-tamas** [vzhd1701/gridplayer#234] [General]
I'd like to launch a previously saved playlist from a command line. I tried running: 
`gridplayer myplaylist.gpls`
`gridplayer --playlist myplaylist.gpls`
`gridplayer --playlist=myplaylist.gpls`
`gridplayer --help`
`gridplayer -help`
but none of these commands work. Is there a way to do this?

Also, if the contents of the file myplaylist.gpls change, will gridplayer notice these changes and load the new streams and/or remove the old streams? If not, how can I add new streams to an already opened gridplayer using the command line?

Thanks for a great app!

**Comment by mutanthost** [comment]
If you are using the Mac version try this:
/Applications/GridPlayer.app/Contents/MacOS/GridPlayer pathtoyourplaylist.gpls
